app/models/rental.py

```diff
@@ -1,4 +1,12 @@
+from sqlalchemy.orm import backref
 from app import db
+from datetime import datetime, timedelta
 
 class Rental(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
\ No newline at end of file
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    video_id = db.Column(db.Integer, db.ForeignKey("video.id"), primary_key=True, nullable=False)
+    customer_id = db.Column(db.Integer, db.ForeignKey("customer.id"), primary_key=True, nullable=False)
+    due_date = db.Column(db.DateTime)
+    checked_out = db.Column(db.Boolean, default = False)
+    #customer = db.relationship("Customer", backref='rentals', lazy=True)
+    #video = db.relationship("Video", backref='rentals', lazy=True)
```

Comment:
These relationships _could_ be included, but since you already have a secondary relationship, they can interfere with one another. In general, I would avoid the secondary relationship in this case, since we're not working with a pure join table for rental, but that would then require looking into cascaded operations to be able to handle the case where a video or customer that was involved in a rental (active or not) is deleted. However, helpers would still be helpful for getting things like the number of active rentals for videos and customers, since this wouldn't include a check for the checked out status (note that there are ways to specify custom joins for the relationships that _could_ take the checked out field into account).